app/src/pages/index.tsx

```diff
@@ -12,21 +19,40 @@ export default function Home() {
                 <title>CL311</title>
             </Head>
             <main>
-                <Container>
-                    <Stack gap={3}>
-                        <PageHeader>CL311 Home Page</PageHeader>
-                        <p>Welcome to CodeDay Labs 311!</p>
-                        <DeleteIcon/><DeleteIcon color="primary" /><DeleteIcon color="secondary" /><DeleteIcon color="success" /><DeleteIcon color="action" /><DeleteIcon color="disabled" />
-                        <p>
-                            While we get set up, take a look at our{' '}
-                            <Link href="/guestbook">guestbook</Link>.
-                        </p>
-                        <p>
-                            Report list under construction {' '}
-                            <Link href="/report">report</Link>.
-                        </p>
-                    </Stack>
-                </Container>
+                <Frame>
+                    <Container>
+                        <Stack gap={3}>
+                            <Grid container justifyContent="center">
+                                <StyledHeading>Welcome to our CodeDay Labs 311!</StyledHeading>
+                                <StyledSubHeading>
+                                    Report Local Issues. Improve Your Community.
+                                </StyledSubHeading>
+                                <StyledSubHeading>
+                                    CL311 makes reporting issues to your city quick, easy and
+                                    effective.
+                                </StyledSubHeading>
+                            </Grid>
+                            <Grid container justifyContent="center">
+                                <StyledButton variant="secondary" href="/report/dashboard">
+                                    <TextBtn>Dashboard</TextBtn>
+                                </StyledButton>
```

Comment:
This is called a "call to action" / CTA, and it's the primary action we want the primary customer to take when entering the page. i think "dashboard" is mostly for city employees, do you think the button should be "report an issue"?